### DISCUSSION TITLE
Invoice expired

### DISCUSSION THREAD

**Discussion by Dcclarke1949** [btcpayserver/btcpayserver#4912] [Technical Support] [answered]
I've tried to transfer but get 599 minutes then says invoice expired 
This happened 12 time's

**Comment by pavlenex** [comment]
If you're trying to pay to bitcoin bonus, that's a scam don't pay it. https://blog.btcpayserver.org/bitcoin-bonus-scam/

**Reply by Dcclarke1949** [reply, marked as answer]
Thank you
